code/controllers/configuration/entries/jobs.dm

```diff
@@ -0,0 +1,185 @@
+// This file pretty much just handles all of the interactions between jobconfig.toml and the codebase. This is started by work originating in SSconfig, so I'm okay with it being here.
+
+/// Initializes all of the config singletons for each job config type and adds it to the `job_config_datum_singletons` var list.
+/datum/controller/subsystem/job/proc/generate_config_singletons()
```

Comment:
the doc here isn't quite right anymore since it was changed to just return a value